scanpipe/pipes/symbolmap.py

```diff
@@ -93,36 +100,48 @@ def map_resources_with_symbols(
         logger(f"No mappings using {map_type} for: {to_resource.path!r}")
 
 
-def match_source_symbols_to_binary(source_symbols, binary_symbols):
-    binary_symbols_set = set(binary_symbols)
+def match_source_symbols_to_deployed(
+    source_symbols,
+    deployed_symbols,
+    matching_ratio,
+    matching_ratio_small_file,
+    small_file_threshold=SMALL_FILE_SYMBOLS_THRESHOLD,
+):
+    deployed_symbols_set = set(deployed_symbols)
     source_symbols_set = set(source_symbols)
     source_symbols_count = len(source_symbols)
     source_symbols_unique_count = len(source_symbols_set)
 
     source_symbols_counter = Counter(source_symbols)
 
-    common_symbols = source_symbols_set.intersection(binary_symbols_set)
+    common_symbols = source_symbols_set.intersection(deployed_symbols_set)
     common_symbols_count = sum(
         [source_symbols_counter.get(symbol) for symbol in common_symbols]
     )
-    common_symbols_ratio = common_symbols_count / source_symbols_count
+    common_symbols_ratio = 0
+    if common_symbols_count > 0:
+        common_symbols_ratio = common_symbols_count / source_symbols_count
+
     common_symbols_unique_count = len(common_symbols)
-    common_symbols_unique_ratio = (
-        common_symbols_unique_count / source_symbols_unique_count
-    )
+    common_symbols_unique_ratio = 0
+    if common_symbols_unique_count > 0:
+        common_symbols_unique_ratio = (
+            common_symbols_unique_count / source_symbols_unique_count
+        )
+
     stats = {
         "common_symbols_unique_ratio": common_symbols_unique_ratio,
         "common_symbols_ratio": common_symbols_ratio,
     }
 
     if (
-        common_symbols_ratio > MATCHING_RATIO_RUST
-        or common_symbols_unique_ratio > MATCHING_RATIO_RUST
+        common_symbols_ratio > matching_ratio
+        or common_symbols_unique_ratio > matching_ratio
     ):
         return True, stats
-    elif source_symbols_count > SMALL_FILE_SYMBOLS_THRESHOLD and (
-        common_symbols_ratio > MATCHING_RATIO_RUST_SMALL_FILE
-        or common_symbols_unique_ratio > MATCHING_RATIO_RUST_SMALL_FILE
+    elif source_symbols_count > small_file_threshold and (
+        common_symbols_ratio > matching_ratio_small_file
+        or common_symbols_unique_ratio > matching_ratio_small_file
```

Comment:
Could we wrap those conditions in a explicit variable name?

Reply:
I should remove this. This change is no longer needed since we're not using `match_source_symbols_to_binary` for javascript symbol matching.